tests/pipelines/stable_diffusion/test_stable_diffusion_img2img.py

```diff
@@ -106,7 +106,7 @@ def get_dummy_inputs(self, device, seed=0):
             "generator": generator,
             "num_inference_steps": 2,
             "guidance_scale": 6.0,
-            "output_type": "numpy",
+            "output_type": "np",
```

Comment:
This is a breaking change so we need to pass a deprecation cycle here (see other comments).

Also let's now add more tests here.